karma.conf.js

```diff
@@ -46,6 +46,15 @@ const newWebpackConfig = Object.assign({}, webpackConfigProd, {
     }, {
       test: /\.svg$/,
       loader: 'url?limit=10000&mimetype=image/svg+xml',
+    }, {
+      test: /\.jpg$/,
+      loader: 'url?limit=10000&mimetype=image/jpeg',
+    }, {
+      test: /\.webm$/,
```

Comment:
I think for the videos you could use the file-loader directly since these should never be inlined. The url-loader hands-off to the file-loader if the size is over the limit param.